Management command to download OSM extracts for relation ids

smn opened the pull request:
This accompanies the `import_geojson` management command. It takes OSM relation ids as positional arguments and then downloads the relevant files from a GitHub repository. It defaults to `nyaruka/posm-extracts`.

rowanseymour commented on temba/locations/management/commands/download_geojson.py:
```diff
@@ -0,0 +1,70 @@
+        parser.add_argument('--oauth-token',
+                            dest='oauth_token',
+                            default=None,
+                            help='The OAuth token to use when authenticating '
```
max-line-length=120 ...

rowanseymour commented on temba/locations/management/commands/download_geojson.py:
```diff
@@ -0,0 +1,70 @@
+import os
```
`from __future__ import print_function, unicode_literals`

rowanseymour commented on temba/locations/management/commands/download_geojson.py:
```diff
@@ -0,0 +1,70 @@
+import os
+import requests
+import re
```
We've been switching to `regex` instead of `re`

smn commented:
@rowanseymour thanks, updated.

The reason I added this was so that we could automate the geojson stuff using environment variables in a docker deploy. If not of interest here I can ship it as part of the docker builds.

smn commented:
@nicpottier 👍  👎  ?

nicpottier commented:
Sorry, tried to comment on this over email but apparently didn't work.

This is definitely super cool, note that the Nyaruka extracts are definitely stale but this is a nice bootstrap. Someday when we have lots of spare time I really want to build these automatically and publish them on a fancy site, but ya, not a lot of that.
